### PR TITLE
Improvements around seal workers and miner setup

### PR DESCRIPTION
**Pull request by hsanjuan**

### CODE REVIEW

File: docs/mine/lotus/seal-workers.md [hunk 1]
```diff
@@ -1,26 +1,43 @@
...
+If the worker process has only 1 available CPU-core (because of CPU-affinity configuration or `GOMAXPROCS=1`), then it will only be able to run 1 _PreCommit1_ task as there are no resources available to utilize its second _window_.
+
+Note that, at all moments, the miner controls the total number of sectors that can be sealing at the same time via `MaxSealingSectors` and/or `MaxSealingSectorsForDeals` in the Miner's `config.toml`.
+
+All of the above serves to setup custom _sealing pipelines_ where available hardware can be utilized in the most performance fashion, for example, by having co-located workers on a single machine to perform _PreCommit1_ only, and then having a dedicated GPU worker for the _PreCommit2_ and _Commit_ phases on a different hardware, leaving the miner to do the rest of the operations. The final setup will depend on the available hardware and its specifications.
+
+::: callout
+Remember during sealing, significant amounts of data are moved/copied accross workers, so good network connectivity among them is a must.
+:::
+
```
Comment (hsanjuan) [inline docs/mine/lotus/seal-workers.md:33]: @lanzafame I hope I have understood this correctly.

File: docs/mine/lotus/miner-setup.md [hunk 2]
```diff
@@ -119,7 +119,7 @@ You can verify sectors sizes for a network in the [network dashboard](https://ne
 
 To summarize all of the above, make sure that:
 
-- A BLS address exists and has some funds so that he miner can be initialized
+- A BLS address exists and has some funds so that he miner can be initialized.
```
Comment (johnnymatthews) [inline docs/mine/lotus/miner-setup.md:122]:
```suggestion
- A BLS address exists and has some funds so that the miner can be initialized.
```

**Review by johnnymatthews — APPROVED** [review]

File: docs/get-started/lotus/installation.md [hunk 3]
```diff
@@ -133,17 +133,17 @@ This method of building does not produce portable binaries. Make sure you run th
...
 make install-miner-service
 ```
 
-Once installed, you should be able to control Lotus using `systemctl [start|stop] lotus-daemon`.
+::: warning
+Provided service files should be **inspected and edited** according to user needs as they are very generic and may lack specific environment variabes and settings needed by the users.
 
-::: tip
-By default, the `lotus-daemon` service file redirects the logging output to `/var/log/lotus/daemon.log`, so `journalctl` displays nothing.
+One example is that logs are redirected to files in `/var/log/lotus` by default and not visible in `journalctl`.
```
[the remark this answers is not in the document]
Comment (hsanjuan) [reply docs/get-started/lotus/installation.md:146]: Gonna put that in advanced.

**Comment by hsanjuan** [comment]
I think I got everything. Any other fixes to be addressed post-merge.

**Pull request merged**